contracts/base/LendingPool.sol

```diff
@@ -72,6 +72,13 @@ contract LendingPool is Base, LendingPoolInterface {
         whenNotPaused()
         whenLendingPoolNotPaused(address(this))
     {
+        uint256 maxTotalValueLocked = _getSettings().getPlatformSettingValue(
+            _getSettings().consts().MAXIMUM_TOTAL_VALUE_LOCKED()
```

Comment:
As Juan mentioned in the other contract. I would change the constant name to refer to the max TLV for the pool (it looks like it refers to the TVL for the platform).

Reply:
Changed